testenv.py

```diff
@@ -48,13 +46,22 @@ def build(options):
         kdesrcbuild.build.main()
 
 def start(options):
-    print("start " + options.dataset + options.clientconfigset)
     dataset = options.dataset
     clientconfigset = options.clientconfigset
-    container = startContainer("{}:{}".format(settings.REPOSITORY, settings.populatedTag(dataset)), lambda: kolabpopulated.run.main(dataset))
-    kontact.run.main(container, clientconfigset)
-    sh.docker.kill(container)
-    sh.docker.rm(container)
+    standalone = clientconfigset is None
+    if standalone:
+        print("start " + dataset + " in background")
+    else:
+        print("start " + dataset + " " + clientconfigset)
+
+    cname = "{}:{}".format(settings.REPOSITORY, settings.populatedTag(dataset))
+    started = dockerutils.findContainer(cname) != ""
+    container = startContainer(cname, lambda: kolabpopulated.run.main(dataset, standalone))
+    if not standalone:
+        kontact.run.main(container, clientconfigset)
+        if not started:
```

Comment:
The idea is:

if the server is not started we start it and kill it afterwards
the started bool indicates if the server was already started.

so "not started" is true if the server was not running before.